components/support/utils/src/test/java/mozilla/components/support/utils/DownloadUtilsTest.kt

```diff
@@ -4,11 +4,11 @@
 
 package mozilla.components.support.utils
 
+import org.junit.Assert.assertEquals
 import org.junit.Test
 import org.junit.runner.RunWith
 import org.robolectric.RobolectricTestRunner
 
-import junit.framework.Assert.assertEquals
 
```

Comment:
ktlint is failing, because of the addtional empty new line (12). Can you please remove. Thanks for your help!

Reply:
Done :)